### PR TITLE
Issue 3672 radia compute job names

### PR DESCRIPTION
**Pull request by mkeilman**
Notes:

- Most of this is changing "geometry" to "geometryReport" and "solver" to "solverAnimation"
- Fixed a bug where ```template_common.write_dict_to_h5()``` was still wrapped in ```with...as```
- A little renaming to align with work for #2598

### CODE REVIEW

File: sirepo/template/radia.py
Comment (robnagler) [inline sirepo/template/radia.py:585]: This is a pattern which we may want to encapsulate? Returning `error` should be known by something, e.g. http_reply. I think we may want more context, e.g. which simulation. "Radia" is not necessary in the error message as it it is coming out of template.radia and the line/function will be known

File: sirepo/template/radia.py
Comment (robnagler) [inline sirepo/template/radia.py:808]: let's continue to use `pkdlog('file {} not found in {}', filename, p)`. Also, `p` may not be exist if the exception happens inside `_get_sim_file` so you should have a `p = None` outside the `try`

File: sirepo/template/radia.py
Comment (robnagler) [inline sirepo/template/radia.py:811]: This is too course of an error. I don't see where `KeyError` is being thrown, but it might be thrown in any of the functions. Better to wrap this deeper in the code, I think.

File: sirepo/template/radia.py
Comment (robnagler) [inline sirepo/template/radia.py:813]: We've been tending to a model of messages like this now: `'h5path={} not found in file={}'. It's slightly more structured, and it uses names of variables so easier to establish context when debugging.

**Review by robnagler — CHANGES_REQUESTED** [review]
Looks good from the rename perspective. I didn't test it, but assume you did. The comments are unrelated to the name change, just saw some code that has some stylistic and other issues. If you see other code that's similar, fix it up, too. f-strings shouldn't be used in pkdlog especially in exceptions.

File: sirepo/template/radia.py
Comment (mkeilman) [reply sirepo/template/radia.py]: 575512a

I wanted to distinguish between "sirepo radia" and "oleg Radia" - now using "backend" to clarify that

File: sirepo/template/radia.py
Comment (mkeilman) [reply sirepo/template/radia.py]: 7679250

Added ```NoH5PathError``` which seems general enough to put in ```template_common```

File: sirepo/template/radia.py
Comment (mkeilman) [reply sirepo/template/radia.py]: 7679250

File: sirepo/template/radia.py
Comment (mkeilman) [reply sirepo/template/radia.py]: 7679250

**Review by e-carlin — APPROVED** [review]

**Comment by e-carlin** [comment]
I'll merge after I merge #3661

**Comment by e-carlin** [comment]
@mkeilman there are conflict resulting from merging #3661 . Can you resolve them?

**Comment by mkeilman** [comment]
> @mkeilman there are conflict resulting from merging #3661 . Can you resolve them?

All conflicts should now be resolved

**Comment by mkeilman** [comment]
@e-carlin did the solve work on NERSC?

**Pull request merged**
